src/plays/tic-tac-toe-game/TicTacToeGame.jsx

```diff
@@ -0,0 +1,32 @@
+import { getPlayById } from 'meta/play-meta-util';
+
+import PlayHeader from 'common/playlists/PlayHeader';
+
+import React from "react";
+
+import Game from "./components/Game.jsx"
+
+import "./styles/tic.css";
+
+function TicTacToeGame(props) {
+  // Do not remove the below lines. 
+  // The following code is to fetch the current play from the URL
+  const { id } = props;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+  return (
+    <>
+      <div className="play-details">
```

Comment:
Add a unique class name here like,

```css
<div className="play-details tic-tac-toe-tj">
```

Now use the unique class name `tic-tac-toe-tj` to prefix your CSS styles. It will make sure the least chance of conflict.